src/mediapipe_calculators/calculators.md

```diff
@@ -0,0 +1,15 @@
+# Mediapipe calculators {#ovms_docs_mediapipe_calculators}
+
+## Introduction
+
+"Each calculator is a node of a graph. The bulk of graph execution happens inside its calculators. Ovms has its own calculators but can also use newly developed calculators or reuse the existing calculators defined in the original mediapipe repository."
```

Comment:
```suggestion
"Each calculator is a node of a graph. The bulk of graph execution happens inside its calculators. OpenVINO Model Server has its own calculators but can also use newly developed calculators or reuse the existing calculators defined in the original mediapipe repository."
```

This readme most likely will be placed inside Mediapipe repository so the acronym could not be easily understandable there.